go-backend/db/common_test.go

```diff
@@ -0,0 +1,30 @@
+package db
+
+import (
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/jmoiron/sqlx"
+	logger "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+func InitMockDB() (s Storer, sqlConn *sqlx.DB, sqlmockInstance sqlmock.Sqlmock) {
+	mockDB, sqlmock, err := sqlmock.New()
+	if err != nil {
+		logger.WithField("error in mock init", err).Error("error occurred")
```

Comment:
inside WithField it should be "err": err.Error()

the message in `Error` can be `error initialising mock db`